bridge-history-api/internal/logic/l2_event_parser.go

```diff
@@ -113,12 +123,31 @@ func (e *L2EventParser) ParseL2EventLogs(logs []types.Log, blockTimestampsMap ma
 			event := backendabi.L2SentMessageEvent{}
 			err := utils.UnpackLog(backendabi.IL2ScrollMessengerABI, &event, "SentMessage", vlog)
 			if err != nil {
-				log.Warn("Failed to unpack SentMessage event", "err", err)
+				log.Error("Failed to unpack SentMessage event", "err", err)
 				return nil, nil, err
 			}
+			from := event.Sender.String()
```

Comment:
the code looks the same in l1 fetcher, could use function to wrap the code?

Reply:
fixed in [07426cf](https://github.com/scroll-tech/scroll/pull/1088/commits/07426cff4d7afb357c5d13aa1e4b7e7a8136e87d)